pkgs/desktops/gnome-3/apps/gnome-boxes/default.nix

```diff
@@ -24,6 +24,8 @@ in stdenv.mkDerivation rec {
     meson ninja vala pkgconfig gettext itstool wrapGAppsHook gobjectIntrospection desktop-file-utils
   ];
 
+  propagatedUserEnvPkgs = [ spice-gtk ];
```

Comment:
Could you add a comment mentioning that it is needed for installing the polkit rules